sdk/rt/speechmatics/rt/_async_multi_channel_client.py

```diff
@@ -21,8 +20,6 @@
 from ._models import TranscriptionConfig
 from ._models import TranslationConfig
 from ._utils.audio import b64_encode_audio
-from ._utils.message import build_start_recognition_message
-from .constants import CHUNK_SIZE
 
 
 class AsyncMultiChannelClient(_BaseClient):
```

Comment:
Do we have a similar feature in multi-channel or is that stream only?

Reply:
eventually multi-chan will require a similar implementation, but given that multi chan feature hasn't been released yet I will leave it alone

Reply:
Yep, makes sense - c r i t i c a l  p a t h